pipeline/config/exoscale/rook/values.yaml

```diff
@@ -0,0 +1,105 @@
+# Values shared with all clusters
+commons:
+  podSecurityPolicies:
+    enabled: false
+  networkPolicies:
+    enabled: true
+    dnsPeers:
+      - cidr: 0.0.0.0/0
+  cluster:
+    preset: "none"
+    mon:
+      resources:
+        requests:
+          cpu: 400m
+          memory: 500Mi
```

Comment:
Question should we have any kind of resource limit on ```mon:``` and ```osd:```or is it fine without?

Reply:
I don't think we need to override it, the limits do get set to the chart defaults. I lowered the requests in order to make sure that there are enough resources for everything to get scheduled.